app/components/chart-donut.js

```diff
@@ -43,6 +43,14 @@ export default Component.extend({
       return;
     }
 
+    if (!isIcon) {
+      svg.classed("svg-container", true)
```

Comment:
Yes, but.. It should actually be added to a parent div or other container.  Wasn't sure if that is the route we want to go, so I added it to the svg element.  Please let me know you thoughts about that.